src/main/cpp/src/loader/tiledb_loader.cc

```diff
@@ -475,6 +476,9 @@ void VCF2TileDBLoader::common_constructor_initialization(
   assert(m_num_orders_owned != 0u);
   m_max_size_per_callset = m_per_partition_size/m_num_orders_owned;
   //Converter processes run independent of loader when num_converter_processes > 0
+  if(g_show_import_progress){
```

Comment:
Most of the changes do not seem to be covered by tests, see CodeCov's coverage details - https://github.com/GenomicsDB/GenomicsDB/pull/176/checks?check_run_id=3404564341. Would it be possible to add tests to test_tools.sh for vcf2genomicsdb specifically? Feel free to refactor test_tools.sh as needed.